Added update functionality

JethroWarnett opened the pull request:
Added the functionality of updating the user attributes. At the moment this functionality is very basic and one may need to add updating assertions and constraints

JethroWarnett committed:
Added update functionality

Added the functionality of updating the user attributes. At the moment this functionality is very basic and one may need to add updating assertions and constraints

JethroWarnett committed:
Update db.sqlite

dominiqueroux reviewed (changes requested):
return value is not nice in my opinion (just gives a one)
i would return the new user object

Commit:
checkout db from master branch so the merge conflict resolves

JethroWarnett committed:
Merge branch 'UserAttributesUpdate' of https://github.com/scg-unibe-ch/ese2020-team5 into UserAttributesUpdate

dominiqueroux committed:
Merge branch 'master' into UserAttributesUpdate

JethroWarnett committed:
Merge branch 'UserAttributesUpdate' of https://github.com/scg-unibe-ch/ese2020-team5 into UserAttributesUpdate

JethroWarnett committed:
Refactored Update Method in UserService

Changed the method update in UserService, so that  the changed UserAttributes are returned
